src/components/file-upload.tsx

```diff
@@ -270,8 +270,8 @@ class FileUpload extends React.Component<FileUploadProps, FileUploadState> {
     if (reader.target === null || reader.target.result === null) return
     //Returns first page of document
     var combinedValues = [];
+    let earningsPage = "";
```

Comment:
Can we either:
- use revert that line so it is `undefined` for earningsPage initial setting and then check for `undefined`, instead of `""` in the if statement on 291, 
- or set it to undefined in the if statement?

<img width="398" alt="image" src="https://user-images.githubusercontent.com/283343/82947426-0823bc00-9f6e-11ea-917d-f0abf7533838.png">

"" appears as 0 under the upload button I think.

Reply:
This didn't fix it after all, hmm. Tricky.